src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/coreperson/CorePersonMappingResource.kt

```diff
@@ -120,6 +120,38 @@ class CorePersonMappingResource(private val service: CorePersonService) {
   ): Page<CorePersonMappingDto> =
     service.getCorePersonMappingsByMigrationId(pageRequest = pageRequest, migrationId = migrationId)
 
+  @GetMapping("/person/nomis-prison-number/{nomisPrisonNumber}")
+  @Operation(
+    summary = "Get person mapping by nomis prison number",
+    description = "Retrieves the person a mapping by Nomis Prison Number (Offender number). Requires role ROLE_NOMIS_CORE_PERSON",
```

Comment:
bad wording - will update in next pr - should read 'Retrieves the core person mapping'